index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="menu-bar">
+    <div><a class="name" href="#">Eyob yoseph</a></div>
+    <nav>
+      <img class="humburger" src="img/Icon - Menu.png" alt="">
+    </nav>
+  </header>
+  <main>
+    <section class="intro-page">
+     <h1 class="greeting"> Hey There.<span class="span-text">I’m Eyob</span></h1>
+     <h3 class="job">I’am a Software Developer</h3>
+     <p class="job-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+      <ul class="social-links">
+        <li><a href="#"><img class="link-image" src="img/Icon -GitHub.png" alt=""></a></li>
+        <li><a href="#"><img class="link-image" src="img/Icon - Linkedin.png" alt=""></a></li>
+        <li><a href="#"><img class="link-image" src="img/Icon - Angelist.png" alt=""></a></li>
+        <li><a href="#"><img class="link-image" src="img/Icon - Twitter.png" alt=""></a></li>
+        <li><a href="#"><img class="link-image" src="img/Icon - Medium.png" alt=""></a></li>
+      </ul>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the `li` decorations are removed. You can consider using `text-decoration: none` to ensure that the design matches the Figma design
- [ ] [OPTIONAL] Additionally, the required `alt` attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader) ☺️ . I suggest that you provide a descriptive `alt text` 👍 .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219874782-cc9caa38-0743-4601-ac53-c6a75e5c416e.png)